utils/dialog/DEPENDS

```diff
@@ -1 +0,0 @@
-depends  ncurses
```

Comment:
Sorry but that is just wrong. If a module has a dependency it should be listed in DEPENDS. Doesn't matter if it comes pre-installed or not :)